sktime/utils/datetime.py

```diff
@@ -13,8 +18,7 @@
 
 
 def _coerce_duration_to_int(duration, freq=None):
-    """Coerce durations into integer representations for a given unit of
-    duration
+    """Coerce durations into integer representations for a given unit of duration.
```

Comment:
the description for `freq` is incomplete, can you add this? I know it's not you who wrote it, but while we're working on this.

Reply:
Happy to. Will complete the description of 'freq'.

Reply:
should include what it can be - I think it should be `valid pandas.datetime freq string`?

Reply:
(I think `week` etc are not valid strings here)

Reply:
I see. how about "Frequency of the above duration type"?